tests/wodan-irmin/test_wodan.ml

```diff
@@ -29,7 +29,7 @@ module DB_ram =
 
 (* let store = Irmin_test.store (module Wodan_irmin.Make(DB_ram)) (module Irmin.Metadata.None) *)
 let store =
-  (module Wodan_irmin.KV_chunked (DB_ram) (Irmin.Contents.String)
+  ( module Wodan_irmin.KV_chunked (DB_ram) (Irmin.Contents.String)
```

Comment:
This does not seem right to me, reported in https://github.com/ocaml-ppx/ocamlformat/issues/722